Networking/Networking/Network/ApplicationPasswordNetwork.swift

```diff
@@ -0,0 +1,47 @@
+import Combine
+import Foundation
+import Alamofire
+
+public class ApplicationPasswordNetwork: Network {
```

Comment:
Could you add a comment explaining why this network is necessary?

Reply:
Done in b2f3d741f6009f82b03a4e7fcc0ab51da3ca977d